lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -367,7 +367,7 @@ def fetch_course_info_with_queries(self, course, sql_queries, mongo_queries):
         self.assertEqual(resp.status_code, 200)
 
     def test_num_queries_instructor_paced(self):
-        self.fetch_course_info_with_queries(self.instructor_paced_course, 23, 4)
+        self.fetch_course_info_with_queries(self.instructor_paced_course, 24, 4)
```

Comment:
@bradenmacdonald It is. I don't know why either -- it was failing, so I just had to change it. Not sure why it is this way.